perun/utils/common/cli_kit.py

```diff
@@ -832,6 +834,8 @@ def set_optimization(_: click.Context, param: click.Argument, value: str) -> str
     :param value: value of the parameter
     :return: the value
     """
+    from perun.collect.trace.optimizations.optimization import Optimization
```

Comment:
Is this intended? Again it looks automatic and imports in functions are shuned by the linting.

Reply:
This is actually intended. `Optimization` is a global object in the `optimizations` module and it is too complicated to refactor it right now, as many more modules and packages would have to be lazy imported. So this is a temporary solution, similarly to how many other packages do import their nested modules in `lazy_get_cli_commands`, e.g., `view`, `collect`, etc.

Reply:
Add fixme there then, so we do not forget.

Reply:
There are many more places where local imports are used in the codebase right now before we decided to adopt the `lazy-loader` approach, so this is something I definitely intend to cover in subsequent PRs regardless. However, for peace of mind, I added a TODO comment there.